modules/gob-web/modules/student/student-summary.js

```diff
@@ -27,13 +27,14 @@ const welcomeMessages = [
 	'Hello, ',
 	'こんにちは、', // japanese
 	'ようこそ、', // japanese
-	'Fram! Fram! ',
-	'Salut, ',
+	'Fram! Fram! ', // new norwegian
```

Comment:
should we add proper norwegian?

should we change this to "olaf"?

Reply:
[source](https://en.wikipedia.org/wiki/St._Olaf_College#Seal_and_motto)
> The motto Fram! Fram! Kristmenn, Krossmenn, written in New Norwegian

Reply:
I'd also accept [Nynorsk](https://en.wikipedia.org/wiki/Nynorsk)